awx/main/wsrelay.py

```diff
@@ -306,7 +306,8 @@ async def run(self):
         database_conf['OPTIONS'] = deepcopy(database_conf.get('OPTIONS', {}))
 
         for k, v in settings.LISTENER_DATABASES.get('default', {}).items():
-            database_conf[k] = v
+            if k != 'OPTIONS':
```

Comment:
Does this mean the connection we are building here would not get the tcp settings ? https://github.com/ansible/tower/blob/devel/awx/settings/defaults.py#L41-L50

If that is the case then we can’t do an overwrite, we need to do a merge of the options.

OR

we do nothing. Cant the user just fill out the listener connection just like the default connection?

Reply:
@chrismeyersfsu it gets OPTIONS in the next line of the code, for some reason it was written before to duplicate.